### DISCUSSION TITLE
Clean up VB Code in Mircosoft.VisualBasic.Form

### DISCUSSION THREAD

**Discussion by paul1956** [dotnet/winforms#10090] [Ideas] [answered]
There are ~67-85 (depending on where I start) code suggestions in the Mircosoft.VisualBasic.Form project. I would like to do a PR to clean them up. None change code but I don't see how to open this as an issue and don't want to do it if the PR would not be accepted.

**Comment by elachlan** [comment, marked as answer]
I see you created the PR at #10117 

So I will close this.
